movie/views.py

```diff
@@ -1,3 +1,80 @@
-from django.shortcuts import render
+from operator import itemgetter
 
-# Create your views here.
+from django.db.models import Avg
+from django.http      import JsonResponse
+from django.views     import View
+
+from .models import (
+        Category, Movie, MainSort, MovieImage, 
+        Gallery, MovieLike, UserStar, Panel, Actor,
+        Comment, Genre, Collection, MovieGenre, CommentLike
+        )
+
+
+class MovieListView(View):
+    def get(self, request):
+        try:
+            results = {}
+
+            box_office        = MainSort.objects.get(name="박스오피스")
+            box_office_movies = Movie.objects.filter(main_sort=box_office)
```

Comment:
이 부분은 한 줄로도 표현이 가능합니다.
```suggestion
            box_office_movies = Movie.objects.filter(main_sort  = Mainsort.objects.get(name="박스오피스"))
```

Reply:
추가적으로 해당 부분을 효율적으로 리팩토링 하기 위해서 select_related 와  prefetch_related 를 알아보시면 좋을 듯 합니다:)

Reply:
네 한 번 알아보겠습니다!